dist/index.js

```diff
@@ -54,6 +57,7 @@ var Dialog = createReactClass({
   getDefaultProps: function () {
     return {
       closeButtonLabel: 'Close this dialog window',
+      closeButtonContent: '\u00D7',
```

Comment:
I think it is :) 
http://www.fileformat.info/info/unicode/char/00d7/index.htm
(https://shripadk.github.io/react/docs/jsx-gotchas.html)